docs/src/installation_instructions.md

```diff
@@ -1,9 +1,5 @@
 # Installation instructions
 
-You can install the latest version of ParameterEstimocean.jl via the built-in
-package manager (by pressing `]` in the Julia REPL command prompt) to add the package and also to 
-instantiate/build all the required dependencies
-
```

Comment:
why delete these?

We have registered the package but there is a 3 day period before it gets merged to the main Julia registry...

https://github.com/JuliaRegistries/General/pull/59267

Reply:
oh I see... it's duplicate!